gatk-sv/test/Module00a/CollectCoverage/workflow.wdl

```diff
@@ -0,0 +1,30 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/broadinstitute/gatk-sv/v0.15-beta/wdl/Structs.wdl"
+import "https://raw.githubusercontent.com/broadinstitute/gatk-sv/v0.15-beta/wdl/CollectCoverage.wdl" as cov
```

Comment:
Although this is convenient, linking to an external resource makes for a less confident approve for running in the analysis runner, because:
- we're relying on an external resource to run our workflows. 
- tags can change

Given it's the same repository, can't we merge changes from upstream?

Although I still don't feel great about approving the _soft_ link to an external resource, I'd budge for linking to the commit hash. (But I'd still prefer performing a merge from upstream)

Alternative:
```suggestion
# from gatk-sv/releases/v0.15-beta
import "https://raw.githubusercontent.com/broadinstitute/gatk-sv/f64d5c8b808eed7ba470132d84a86229c4c2cfeb/wdl/Structs.wdl"
import "https://raw.githubusercontent.com/broadinstitute/gatk-sv/f64d5c8b808eed7ba470132d84a86229c4c2cfeb/wdl/CollectCoverage.wdl" as cov
```